how-to-release.md

```diff
@@ -52,29 +60,31 @@ A major release changes either the middle or the most significant digit. In this
   > git push upstream r0.3
 ```
 
-Once the steps above are done, we need to make version changes to both `master` and `r0.3`:
+After the above steps are performed, version changes needs to be updated to both `master` and `r0.3`:
+
 *  In `master`, create an issue and corresponding pull request to change the `connectorVersion` in `gradle.properties` to `0.4.0-SNAPSHOT`. Note that we are bumping up the middle digit because, in our example, we are releasing `0.3.0`. If we were releasing say `1.0.0`, then we would change `connectorVersion` to `1.1.0-SNAPSHOT`.

```

Comment:
Is this version 0.4.0 correct? please check

Reply:
yes

Reply:
ok